portfolio/pom.xml

```diff
@@ -45,7 +45,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>sharondong-step</deploy.projectId>
+          <deploy.projectId>sharondong-step-2020</deploy.projectId>
```

Comment:
Is the new project not working?

Reply:
Yes, it needs a billing account to deploy it as well, so i have been using the old project for now.